src/txrebroadcast.cpp

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <chainparams.h>
+#include <consensus/consensus.h>
+#include <logging.h>
+#include <miner.h>
+#include <script/script.h>
+#include <txrebroadcast.h>
```

Comment:
I think we prefer to include all headers that are used in the translation unit, even if they're also included in the corresponding header file. You don't need to remove the "#include <util/time.h> in the _[rebroadcast] Track rebroadcast attempts_ commit.